.github/workflows/lsif-typescript.yml

```
@@ -0,0 +1,26 @@
+name: upload-index
+on:
+  push:
+    paths:
+      - '**.ts'
+
+jobs:
+  upload-index:
+    if: github.repository == 'sourcegraph/lsif-typescript'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install npm dependencies
+        run: yarn --ignore-engines --ignore-scripts
+      - name: Install src-cli
+        run: |
+          curl -L https://sourcegraph.com/.api/src-cli/src_linux_amd64 -o /usr/local/bin/src
+          chmod +x /usr/local/bin/src
+      - name: Install lsif-typescript
+        run: yarn run build
```

We can run the TS files directly with ts-node. 
```suggestion
        run: ./node_modules/.bin/ts-node src/main.ts index --no-progress-bar
```
We may want to tweak the default value of `--no-progress-bar` so that it's automatically configured in CI environments. You can typically achieve this by either detecting the env variable `CI=true` or by testing for the presence of tty.

---

The benefit of `ts-node` is that it prints out stack traces from the original TS files instead of the compiled JS output.

---

Updated.